README.md

```diff
@@ -41,6 +41,46 @@ The Image Tool provides three functions within the main script:
 
 Use the [Quick Start](site/quickstart.md) guide to create a Linux based WebLogic Docker image.
 
+## Building From Source
+
+The Image Tool installer is available for download on the [Releases](releases) page.  If you wish
+to build the installer from source instead of downloading it, please follow these instructions:
+- Download and install JDK 8u261+
+- Download and install Maven 3.6.3+
+- Clone this repository to your local environment using one of the options under `Code` near the top of this page.
+- From inside the top-level directory of the cloned project, `weblogic-image-tool`, using Maven, execute one or 
+more of these phases:
+    - `validate` - validate the project is correct and all necessary information is available.
```

Comment:
validate -> Validate (capitalize the first word of each bulleted item description)